code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -14,6 +22,7 @@
 	attack_type = BURN //burn bish
 	exotic_bloodtype = "E"
 	damage_overlay_type = "" //We are too cool for regular damage overlays
+	species_age_max = 300
```

Comment:
unrelated, but we really need to add some handling to guestbook age code that bases the age off of this variable instead of how it works now.